src/controllers/planterRegistration.controller.ts

```diff
@@ -10,7 +10,7 @@ export class PlanterRegistrationController {
     public planterRepository: PlanterRegistrationRepository,
   ) {}
 
-  @get('/planter-registration', {
+  @get('/organization/{organizationId}/planter-registration', {
```

Comment:
I think we'll need to keep the original endpoint for non-org requests.
The `trees` and `planter` org endpoints are implemented in separate controllers: `treesOrganization.controller.ts` and `planterOrganization.controller.ts` respectively.

Reply:
Agreed. I can add the /planter-registration endpoint with the org id to the planterOrganization.controller.ts and keep the non-org endpoint on planter.controller.